src/QA/QA.Web/Controllers/DashboardController.cs

```diff
@@ -17,9 +20,26 @@ public DashboardController(DashboardOrchestrator orchestrator)
         [HttpGet("/", Name = RouteNames.Dashboard_Index_Get)]
         public async Task<IActionResult> Index()
         {
-            var vm = await _orchestrator.GetDashboardViewModelAsync();
+            var vm = await _orchestrator.GetDashboardViewModelAsync().ConfigureAwait(false);
             
             return View(vm);
         }
+
+        /// <summary>
+        /// Opted to use POST instead of GET as it keeps the URL clean of query string and path
+        /// </summary>
+        [HttpPost("/", Name = RouteNames.Dashboard_Index_Post)]
+        public async Task<IActionResult> Index(string searchTerm)
+        {
+            var vm = await _orchestrator.GetDashboardViewModelAsync().ConfigureAwait(false);
+
+            //I would prefer not to go forward if search term is empty
```

Comment:
Don't think we need this comment.  I think its OK to keep the controller dumb and let client handle empty queries.  That's the approach we've taken for validation, etc so fits into this approach.